src/main/java/org/mifos/connector/mpesa/flowcomponents/validation/ValidationWorker.java

```diff
@@ -0,0 +1,72 @@
+package org.mifos.connector.mpesa.flowcomponents.validation;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.camunda.zeebe.client.ZeebeClient;
+import org.apache.camel.CamelContext;
+import org.apache.camel.Exchange;
+import org.apache.camel.ProducerTemplate;
+import org.apache.camel.support.DefaultExchange;
+import org.mifos.connector.common.channel.dto.TransactionChannelRequestDTO;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import java.util.Map;
+
+import static org.mifos.connector.mpesa.camel.CamelProperties.*;
+
+@Component
+public class ValidationWorker {
+
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    private ZeebeClient zeebeClient;
+
+    @Autowired
+    private ProducerTemplate producerTemplate;
+
+    @Autowired
+    private CamelContext camelContext;
+
+    @Value("${zeebe.client.evenly-allocated-max-jobs}")
+    private int workerMaxJobs;
+
+    @PostConstruct
+    public void setupWorkers() {
+
+        zeebeClient.newWorker()
+                .jobType("transactionValidation")
```

Comment:
What is validation here? Shouldn't validation route and worker be in peso-connector?

Reply:
I thought the worker will be here which will be calling the validation endpoint in amp-pesa. I will move the worker to ams-pesa and send the PR.

Reply:
I have removed the validation part from this PR.